lbrynet/wallet/manager.py

```diff
@@ -15,6 +15,12 @@
 log = logging.getLogger(__name__)
 
 
+class ReservedPoints:
+    def __init__(self, identifier, amount):
+        self.identifier = identifier
+        self.amount = amount
+
+
```

Comment:
Why did this get moved? Actually, it would be even better if it was deleted altogether, could you do the necessary refactoring to make that happen? I don't think it's used in many places.